pkg/driverbuilder/builder/debian.go

```diff
@@ -174,11 +178,11 @@ func debianHeadersURLFromRelease(kr kernelrelease.KernelRelease) ([]string, erro
 func fetchDebianHeadersURLFromRelease(baseURL string, kr kernelrelease.KernelRelease) ([]string, error) {
 	extraVersionPartial := strings.TrimSuffix(kr.FullExtraversion, "-"+kr.Architecture.String())
 	matchExtraGroup := kr.Architecture.String()
-	rmatch := `href="(linux-headers-%s\.%s\.%s%s-(%s)_.*(%s|all)\.deb)"`
+	rmatch := `href="(linux-headers-%d\.%d\.%d%s-(%s)_.*(%s|all)\.deb)"`
```

Comment:
Please double check all of this; i should've fixed everything but a careful look is needed!